cypress-tests/cypress/e2e/PaymentUtils/Commons.js

```diff
@@ -98,6 +98,35 @@ const multiUseMandateData = {
   },
 };
 
+const billingAddress = {
+  address: {
+    line1: "1467",
+    line2: "Harrison Street",
+    line3: "Harrison Street",
+    city: "San Fransico",
+    state: "California",
+    zip: "94122",
+    country: "BR",
+    first_name: "john",
+    last_name: "doe",
+  },
+};
```

Comment:
if this is specific to a connector or a payment method, then i would recommend having country field empty and override it in respective fields wherever it is used. 

also, if i'm not asking too much, can you please also apply the same changes to rest of the file where ever `billing` exist and is not `null`.

that way, we can do something like:

```js
billing: {
  ...billingAddress,
  country: DE / GB / BR / IN / US
}
```